Source/Core/Core/PowerPC/Jit64/Jit64_Tables.cpp

```diff
@@ -334,45 +336,33 @@ constexpr std::array<GekkoOPTemplate, 10> s_table63_2{{
     {31, &Jit64::fmaddXX},   // fnmaddx
 }};
 
-// TODO: This can be replaced with:
-//
-//       table.fill(&Jit64::FallbackToInterpreter);
-//
-//       whenever we end up migrating to C++20. Prior to C++20,
-//       std::array's fill() function is, unfortunately, not constexpr.
-//       Ditto for <algorithm>'s std::fill. Thus, this function exists
-//       to bridge the gap.
-template <size_t N>
-constexpr void FillWithFallbacks(std::array<Jit64::Instruction, N>& table)
+constexpr std::array<Jit64::Instruction, 64> s_dyna_op_table = []() consteval
 {
-  for (auto& entry : table)
-  {
-    entry = &Jit64::FallBackToInterpreter;
-  }
-}
-
-constexpr std::array<Jit64::Instruction, 64> s_dyna_op_table = [] {
   std::array<Jit64::Instruction, 64> table{};
-  FillWithFallbacks(table);
+  Common::Fill(table, &Jit64::FallBackToInterpreter);
 
   for (auto& tpl : s_primary_table)
   {
+    ASSERT(table[tpl.opcode] == &Jit64::FallBackToInterpreter);
```

Comment:
If the ASSERT fails, then compilation fails, which is the behavior we want. The message in that case isn't that great, so it might make sense to add some kind of `CONSTEVAL_ASSERT` macro instead, but I don't think it's that important. (`static_assert` didn't seem to behave in a useful manner when I tested it, though I don't remember exactly why.)